tooling/minitest/Cargo.lock

```diff
@@ -75,9 +75,9 @@ checksum = "6a987beff54b60ffa6d51982e1aa1146bc42f19bd26be28b0586f252fccf5317"
 
 [[package]]
 name = "libspecr"
-version = "0.1.15"
+version = "0.1.14"
```

Comment:
This downgrades specr to an earlier version. Why that?

Reply:
Because I'm stupid, did not update my `specr-transpile`. It always reset the required version for gen-minirust.